handsontable/src/shortcuts/__tests__/shortcutManager.spec.js

```diff
@@ -704,4 +704,24 @@ describe('shortcutManager', () => {
       expect(callback).toHaveBeenCalledTimes(0);
     });
   });
+
+  describe('`scope` option', () => {
+    it('should create a context with the `table` scope by default', async() => {
+      handsontable();
+
+      const shortcutManager = getShortcutManager();
+      const gridContext = shortcutManager.addContext('test');
+
+      expect(gridContext.scope).toBe('table');
+    });
+
+    it('should create a context with the `global` scope when the `scope` option is set to `global`', async() => {
```

Comment:
Here are tests that check that the props are assigned, but there is a missing test that checks how that scope affects how the keyboard shortcuts are handled (handled or not when Handsontable is listening).